### PR TITLE
[RFC] Add missing features for template v2

### PR DESCRIPTION
**Pull request by richardhj**
## Description

Changes for #1045.
What blocks are relevant for the templates and which are not?

## Checklist
- [x] Read and understood the [CONTRIBUTING guidelines](CONTRIBUTING.md)
- [ ] Created tests, if possible
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary
- [x] Added myself to the `@authors` in touched PHP files
- [ ] Checked the changes with phpcq and introduced no new issues

### CODE REVIEW

**Comment by zonky2** [comment]
you can also fix this ticket https://github.com/MetaModels/core/issues/644 and delete
```
<?php $strRendersettings = isset($this->settings)? 'settings' : 'view'; ?>
```

**Comment by zonky2** [comment]
one equestion @discordier: its a good time to change the echo to te short code ```<?=```? all Contao core templates have this since 3.3(?)

**Comment by discordier** [comment]
Pretty awesome. Thanks so much @richardhj.

Regarding the short echo, yes we could do this now as we require php >= 5.4.

**Comment by richardhj** [comment]
I replaced the echo tags for all core templates. For reference, this is the regex I used for:
Text to find: `<\?php echo (.+?); ?\?>`
Replace with: `<?= $1 ?>`

**Comment by zonky2** [comment]
Good work!

File: contao/templates/mm_filteritem_radiobuttons.xhtml [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
-    $label = sprintf('<label id="lbl_%s_%s" for="opt_%s_%s">%s%s</label>',
-        $this->urlparam,
-        $count,
-        $this->urlparam,
-        $count,
-        $option['value'],
-        (($this->showCount) ? ' <span class="count">('. $this->count[$option['key']] .')</span>' : '')
-    );
-    $count++;
+	$field = sprintf('<input type="radio" name="%s" id="opt_%s_%s" class="radio%s" value="%s"%s />',
```
Comment (discordier) [inline contao/templates/mm_filteritem_radiobuttons.xhtml]: Please stick to space indention instead of using tabs.

Comment (richardhj) [inline contao/templates/mm_filteritem_radiobuttons.xhtml]: ok, this was taken from the html5 template file. Please have a look at the latest commit, I changed some other indentions too.

**Comment by richardhj** [comment]
@discordier or @stefanheimes Could you please add @stefanheimes as author in the copyright for the author validation?

**Comment by discordier** [comment]
Done in 522a241e

**Pull request merged**